packages/repl/svelte.config.js

```diff
@@ -1,29 +1,11 @@
-import path from 'path';
 import adapter from '@sveltejs/adapter-auto';
 
 /** @type {import('@sveltejs/kit').Config} */
-const config = {
+export default {
 	kit: {
-		adapter: adapter(),
-
-		package: {
-			exports: (file) => file === 'index.js'
-		},
-
-		vite: {
-			resolve: {
-				alias: {
-					'@sveltejs/repl': path.resolve('src/lib'),
-					'@sveltejs/site-kit': path.resolve('../site-kit/src/lib')
-				}
-			},
-			server: {
-				fs: {
-					strict: false
-				}
-			}
-		}
-	}
+		adapter: adapter()
```

Comment:
Spacing looks inconsistent in this file. Please use only tabs

Reply:
Done. should look good now